Database/seed.js

```diff
@@ -1,13 +1,14 @@
 const db  = require('./index.js');
-const fs = require('file-system');
+const fs = require('fs');
 const Funding = require('./Funding.js');
+const path = require('path');
 
 const insertSampleData = function(data) {
   Funding.create(data)
     .then(() => db.disconnect());
 };
```

Comment:
I'm thinking it may be good to have your database related logic in the Funding.js file (similar situation in your server's /api/funding_round route handler). E.g. insertSampleData would be a function within the Funding.js module.